### ISSUE TITLE
Vertical scrolling over Helm docs.

### ISSUE THREAD

**Issue by jmrnilsson** [helm/helm-www#1464]
The docs over at https://helm.sh/docs/intro/cheatsheet/ has a transient error so it doesn’t allow for scrolling up or down on IOS Safari. It’s pretty bad.

Same goes for Firefox on IOS..

**Comment by angellk** [comment]
Thanks for bringing this up - do you have cycles to look at this @jmrnilsson ?

**Comment by z4ce** [comment]
It seems to be something with the way the nav bar is scrolling and the behavior seems to be across all docs. I recorded this in Chrome set to iPhone 12, but it seems to do the same thing in Safari on iOS.

https://user-images.githubusercontent.com/4860914/236893867-52ca0219-5f45-424e-aab1-960a8b53283c.mov

**Comment by jmrnilsson** [comment]
Yeah, thanks for the update while I was away. I concur with z4ce, this applies to Safari/FF on IOS on IPhone 2022 SE and IPhone 11. Probably IPhone 12 as well as written above.

Normally I wouldn’t bother. But it renders the docs unreadable. Annoyingly only occurs transiently.

**Comment by jmrnilsson** [comment]
I cannot se the video for some reason. But it should be pretty self-evident once it occurs.

**Comment by antdking** [comment]
This is also affects Firefox on Android, though looks to hitch less often based on the above video.

Dupe of #1393; appears to impact safari, chromium and Firefox on ios + android

**Comment by scottrigby** [comment]
This should now be fixed since the docusaurus migration. Please open new issues for specific OS/device bugs if you still experience them.
